common/contracts/rust/Cargo.toml

```diff
@@ -17,6 +17,8 @@ opt-level = "z"
 lto = true
 debug = false
 panic = "abort"
+# Ensure safe uint handling https://stackoverflow.com/a/64136471/249801
```

Comment:
This comment sounds scary to me. How about: "Opt into extra safety checks on arithmetic operations"?